assets/javascripts/discourse/templates/components/reviewable-category-expert-suggestion.hbs

```diff
@@ -0,0 +1,33 @@
+<div class="row">
+
+  {{#link-to "user" reviewable.user.username}}
+    {{avatar reviewable.user imageSize="small"}}
+    {{reviewable.user.username}}
+  {{/link-to}}
+  {{i18n "category_experts.review.endorsed_count" count=reviewable.endorsed_count}}
+  {{category-link reviewable.category}}
+</div>
+
+<div class="row">
+
+  <table class="endorsed-by-table">
+    <thead>
+      <tr>
+        <td>{{i18n "category_experts.review.endorsed_by" count=reviewable.endorsed_count}}</td>
+      </tr>
+    </thead>
+    <tbody>
+    {{#each reviewable.endorsed_by as |user|}}
+      <tr>
+        <td class="endorser">
```

Comment:
Since the field in reviewable is `endorsed_by` I suggest we use `endorsed-by` here to keep things the same.